runtime/src/main/java/org/corfudb/util/MetricsUtils.java

```diff
@@ -288,24 +292,14 @@ public static void incConditionalCounter(boolean enabled, @NonNull Counter count
      * return a gauge on direct memory used by netty's PooledByteBufAllocator
      */
     private static Gauge<Long> getNettyPooledDirectMemGauge() {
-        return new Gauge<Long>() {
-            @Override
-            public Long getValue() {
-                return PooledByteBufAllocator.DEFAULT.metric().usedDirectMemory();
-            }
-        };
+        return () -> PooledByteBufAllocator.DEFAULT.metric().usedDirectMemory();
```

Comment:
Even though the new one is cleaner, the old on is easier to understand.

Reply:
I agree. I always use IntelliJ to translate lambda to anonymous class when I don't understand. Modified these to keep a unified code style.